webpack.config.js

```diff
@@ -0,0 +1,134 @@
+const path = require('path');
```

Comment:
it will be better to split it by separate file 
something like : 
webpack.dev.config.js
webpack.prod.config.js
webpack.base.config.js